packages/core/src/components/SimpleStyleScrollables/useSplitContentContainerStyles.ts

```diff
@@ -29,56 +28,18 @@ export const contentContainerStyleNames = [
 ];
 
 export default function useSplitContentContainerStyles(
-  originalStyle: StyleProp<ViewStyle>,
-  measuredWidth?: number,
-  measuredHeight?: number,
-  recalculateSizeDeps: React.DependencyList = []
+  originalStyle: StyleProp<ViewStyle>
 ) {
-  // This temporarily removes contentContainerStyle min sizes whenever some
-  // given dependencies change to allow the list to properly recalculate it's measured size
-  const [tempContentContainerStyle, setTempContentContainerStyle] =
-    React.useState<ViewStyle | null>(null);
-
-  React.useEffect(() => {
-    if (tempContentContainerStyle) {
-      setTempContentContainerStyle(null);
-    }
-    // We only want this to run when measured size changes
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [measuredHeight, measuredWidth]);
-
-  React.useEffect(() => {
-    setTempContentContainerStyle({ minHeight: 0, minWidth: 0 });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, recalculateSizeDeps);
-
   return useDeepCompareMemo<Styles>(() => {
     const flattenedStyle = StyleSheet.flatten(originalStyle);
 
-    let contentContainerStyle = pick(
+    const contentContainerStyle = pick(
       flattenedStyle,
       contentContainerStyleNames
     );
 
-    const leftBorderWidth =
-      flattenedStyle?.borderLeftWidth ?? flattenedStyle?.borderWidth ?? 0;
-    const rightBorderWidth =
-      flattenedStyle?.borderRightWidth ?? flattenedStyle?.borderWidth ?? 0;
-    const topBorderWidth =
-      flattenedStyle?.borderTopWidth ?? flattenedStyle?.borderWidth ?? 0;
-    const bottomBorderWidth =
-      flattenedStyle?.borderBottomWidth ?? flattenedStyle?.borderWidth ?? 0;
-
     // contentContainerStyle should always at least fill the parent to ensure sizing changes reflects properly on component and children.
-    // The measured sizes include borders, so we need to subtract those before applying
-    if (measuredWidth) {
-      contentContainerStyle.minWidth =
-        measuredWidth - leftBorderWidth - rightBorderWidth;
-    }
-    if (measuredHeight) {
-      contentContainerStyle.minHeight =
-        measuredHeight - topBorderWidth - bottomBorderWidth;
-    }
+    contentContainerStyle.flexGrow = 1;
```

Comment:
Parent container in the context of of the `contentContainerStyle` is always the same since this container is an internal child of the scrollable components. This just makes sure that this child always fill the parent size, so that any sizing changes you do via the normal `style` also get reflected on that child container. Let me know if that answers your question or not